io/src/test/java/org/commonjava/maven/ext/io/rest/handler/GAVSchema.java

```diff
@@ -16,47 +16,22 @@
 package org.commonjava.maven.ext.io.rest.handler;
 
 import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.ToString;
 
-import java.util.Arrays;
 import java.util.List;
 import java.util.Map;
 
+@ToString
 public class GAVSchema
 {
     @JsonInclude(value = JsonInclude.Include.NON_EMPTY)
-    public String versionSuffix;
+    public List<Map<String, Object>> artifacts;
```

Comment:
Any reason why these are all public fields instead of using the getter/setter convention?

Reply:
Nothing really...its just test code so kept it easy to access